pkg/features/kube_features.go

```diff
@@ -1043,7 +1043,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	PodDisruptionConditions: {Default: true, PreRelease: featuregate.Beta},
 
-	PodHasNetworkCondition: {Default: false, PreRelease: featuregate.Alpha},
+	PodReadyToStartContainersCondition: {Default: false, PreRelease: featuregate.Beta},
```

Comment:
Will it be better to keep this in Alpha for one release and then graduate to Beta. At Beta stage, I believe the recommendation is to have the Condition name going through an official API review and shifted out of the temporary pkg/kubelet/types/constants.go below?

Reply:
Yea I’ll revert this change.

Reply:
I kept this as Alpha for now.